src/routes/arweave.ts

```diff
@@ -54,8 +55,10 @@ arweaveRouter.post('/chunk', async (req, res) => {
     if (
       result.successCount >= Math.min(MIN_SUCCESS_COUNT, CHUNK_POST_URLS.length)
     ) {
+      metrics.arweaveChunkBroadcastCounter.inc({ status: 'success' });
       res.status(200).send(result);
     } else {
+      metrics.arweaveChunkBroadcastCounter.inc({ status: 'fail' });
       res.status(500).send(result);
```

Comment:
maybe it's correct, let me check

Reply:
no this isn't correct, because of validateStatus attribute of the axios call, all successes are not thrown and vice versa, all failures throw, so this is correct my code